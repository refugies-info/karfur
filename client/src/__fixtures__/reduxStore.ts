import { RootState } from "services/rootReducer";
import { SearchResultsState } from "services/SearchResults/searchResults.reducer";
import { UserFavoritesState } from "services/UserFavoritesInLocale/UserFavoritesInLocale.reducer";
import { activeThemesMock } from "./activeThemes";

const initialMockUserState = {
  user: null,
  admin: false,
  traducteur: false,
  expertTrad: false,
  contributeur: false,
  membreStruct: false,
  userId: "",
  userFetched: false,
  rolesInStructure: [],
};
const initialMockLangueState = {
  langues: [],
  languei18nCode: "fr",
  showLanguageModal: false,
  showLangModal: false,
};

const initialMockTranslationState = {
  translation: {
    initialText: {},
    translatedText: {},
  },
  translations: [],
};

const initialUserFavoritesState: UserFavoritesState = {
  favorites: [],
  showFavoriteModal: false
}

const initialMockSearchReults: SearchResultsState = {
  results: {
    dispositifs: [],
    demarches: [],
    dispositifsSecondaryTheme: []
  },
  query: {
    search: "",
    departments: [],
    themes: [],
    needs: [],
    age: [],
    frenchLevel: [],
    language: [],
    sort: "date",
    type: "all",
  }
}

export const initialMockStore = {
  user: initialMockUserState,
  langue: initialMockLangueState,
  activeDispositifs: [],
  tts: { ttsActive: false, showAudioSpinner: false },
  userStructure: null,
  selectedDispositif: null,
  loadingStatus: {},
  translation: initialMockTranslationState,
  activeStructures: [],
  selectedStructure: null,
  allDispositifs: [],
  allStructures: [],
  users: [],
  userFavorites: initialUserFavoritesState,
  userContributions: [],
  needs: [],
  themes: {
    activeThemes: activeThemesMock,
    inactiveThemes: []
  },
  searchResults: initialMockSearchReults
};
