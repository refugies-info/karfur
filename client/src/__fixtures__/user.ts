import { GetUserInfoResponse, UserStatus } from "api-types";
import moment from "moment";

const date = moment("2019-05-01 12:00:00");
export const testUser: GetUserInfoResponse = {
  username: "test",
  _id: "testObjectId",
  structures: [],
  contributions: [],
  phone: "",
  roles: [],
  email: "",
  selectedLanguages: [],
  status: UserStatus.ACTIVE,
  traductionsFaites: []
};

export const testUserWithRoles: GetUserInfoResponse = {
  ...testUser,
  structures: ["testObjectId"],
  roles: [
    { nom: "Admin", _id: "testObjectId", nomPublic: "Admin" },
    { nom: "Trad", _id: "testObjectId", nomPublic: "Trad" },
    {
      nom: "ExpertTrad",
      _id: "testObjectId",
      nomPublic: "ExpertTrad"
    },
    {
      nom: "Contrib",
      _id: "testObjectId",
      nomPublic: "Contrib"
    }
  ]
};
