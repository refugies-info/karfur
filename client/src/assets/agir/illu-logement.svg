<svg fill="none" height="80" viewBox="0 0 80 80" width="80" xmlns="http://www.w3.org/2000/svg"><g clip-rule="evenodd" fill-rule="evenodd"><path d="m68 13.0002c-.5514 0-1-.4486-1-1s.4486-1 1-1 1 .4486 1 1-.4486 1-1 1zm-27 63c-.5514 0-1-.4486-1-1s.4486-1 1-1 1 .4486 1 1-.4486 1-1 1zm-26-66c-.5514 0-1-.44856-1-.99996 0-.55144.4486-1 1-1s1 .44856 1 1c0 .5514-.4486.99996-1 .99996z" fill="#cacafb"/><path d="m53 44.0002c.5523 0 1 .4478 1 1v10c0 .5523-.4477 1-1 1h-10c-.5523 0-1-.4477-1-1v-10c0-.5522.4477-1 1-1zm-6 7h-3v3h3zm5 0h-3v3h3zm-5-5h-3v3h3zm5 0h-3v3h3zm-7-14c.5523 0 1 .4478 1 1 0 .5523-.4477 1-1 1h-10c-.5523 0-1-.4477-1-1 0-.5522.4477-1 1-1z" fill="#e1000f"/><path d="m29 18.0002c.5523 0 1 .4478 1 1 0 .5523-.4477 1-1 1h-7v2h3c.5128 0 .9355.3861.9933.8834l.0067.1166c0 .5523-.4477 1-1 1h-1v2.722l15.5601-7.62.1077-.0453c.0168-.0056.0254-.0085.034-.0112.0987-.031.1989-.0457.298-.0457.0995 0 .1997.0147.2976.0453l.0136.0045.0212.0071.1077.0453 24.5 12c.4859.238.6944.8193.4705 1.3118l-2.5 5.5c-.2375.5225-.865.7383-1.3737.4725l-1.5367-.8033v4.8876c0 .5523-.4477 1-1 1s-1-.4477-1-1v-5.8216l-18-9.518-18 9.517v23.352h4v-15c0-.5522.4477-1 1-1h10c.5523 0 1 .4478 1 1v15h11c.5523 0 1 .4478 1 1 0 .5523-.4477 1-1 1h-40c-.55228 0-1-.4477-1-1 0-.5522.44772-1 1-1h3l.0003-2.0628c-2.58591-.342-4.0003-2.0686-4.0003-4.9372 0-2.8612 1.37478-6.9856 4.1056-12.4472l.0134-.0268.027-.047.036-.0544c.0113-.0163.0233-.0322.0357-.0479.0124-.0155.0252-.0306.0384-.0453.0115-.0129.0238-.0258.0364-.0384.013-.0131.026-.0253.0392-.0372.0143-.0128.0295-.0257.0451-.038l.0478-.0358.0534-.036.0748-.0404.0118-.0058c.0214-.0104.0434-.02.0656-.0289l.0203-.0079.0206-.0074c.0162-.0056.0322-.0107.0482-.0154.0219-.0066.0441-.0123.0663-.0171l.0197-.004c.0621-.0125.1244-.0188.1863-.0192h.0168c.037.0002.074.0026.1111.007l.0752.0122.0198.0042c.0221.0046.0443.0103.0663.0168l.0239.0075.0237.0079c.0143.0049.0279.01.0416.0154.022.0089.044.0185.0656.029l.0116.0057.0728.0404.0555.0361c.0162.0113.0322.0232.0478.0357.0155.0123.0307.0252.0454.0383.0128.0116.0258.0238.0384.0364.013.0131.0253.026.0371.0392.0129.0144.0257.0295.0381.0451l.0358.0478.0349.0534.027.048.0144.0268c2.7308 5.4616 4.1056 9.586 4.1056 12.4472 0 2.8682-1.414 4.5948-3.9993 4.937l-.0007 2.063h6v-22.417l-1.5367.8033c-.5087.2658-1.1362.05-1.3737-.4725l-2.5-5.5c-.2239-.4925-.0154-1.0738.4705-1.3118l7.1421-3.4989c-.1269-.1676-.2022-.3766-.2022-.6031v-3h-1c-.5523 0-1-.4477-1-1v-4c0-.5522.4477-1 1-1zm36.059 28.0018c.2359.0138.4679.1109.6481.2911 1.5416 1.5416 2.2929 3.7955 2.2929 6.7071 0 2.5869-.6252 4.2985-1.9989 4.8266l-.0011 2.1734h3c.5523 0 1 .4478 1 1 0 .5523-.4477 1-1 1h-14c-.5523 0-1-.4477-1-1 0-.5522.4477-1 1-1h3v-11c0-.5522.4477-1 1-1s1 .4478 1 1v11h4l-.0001-2.173c-1.3744-.5276-1.9999-2.2395-1.9999-4.827 0-2.9116.7513-5.1655 2.2929-6.7071.1802-.1802.4122-.2773.6481-.2911zm-29.059-.0018h-8v14h8zm29.0007 2.6123-.035.0559c-.6383 1.0793-.9657 2.5156-.9657 4.3318 0 2.1012.3766 2.9353.9325 2.9964l.0675.0036.0675-.0036c.5559-.0611.9325-.8952.9325-2.9964 0-1.8478-.3389-3.3023-.9993-4.3877zm-52.0008-5.3189-.1982.428c-1.8716 4.0732-2.8017 7.1812-2.8017 9.2786 0 2.1144.8856 3 3 3l.2074-.0029c1.965-.0575 2.7926-.9532 2.7926-2.9971 0-2.1703-.9958-5.4226-3.0001-9.7066zm20.0001 6.7066c.5523 0 1 .4478 1 1v2c0 .5523-.4477 1-1 1s-1-.4477-1-1v-2c0-.5522.4477-1 1-1zm6.999-28.887-23.187 11.357 1.66 3.653 21.0647-11.0092.038-.0189.0163-.0079.0128-.0052.0102-.0048.058-.0218c.0179-.006.0356-.0116.0535-.0167.0086-.0027.0176-.0052.0267-.0075l.0339-.0078c.0159-.0035.0318-.0066.0479-.0093.0132-.0025.0259-.0044.0386-.006l.0233-.0025c.0153-.0016.0307-.0029.0461-.0038l.016-.0009.0684-.0005.0236.0009c.0191.001.0381.0025.0571.0046.0039.0005.0085.001.0131.0016.0192.0022.0323.0041.0454.0063.0094.0018.0183.0034.0272.0052l.018.0035.014.0031c.0044.0012.0094.0024.0143.0036.0181.0044.0275.007.0369.0097.0074.0023.0153.0047.0231.0071l.0289.0094.0682.0266.0178.0082.0173.0078.032.016 21.0647 11.0092 1.66-3.653z" fill="#000091"/></g></svg>