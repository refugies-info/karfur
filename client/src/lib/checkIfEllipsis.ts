export const checkIfEllipsis = (element: HTMLElement) => element.offsetWidth < element.scrollWidth;
