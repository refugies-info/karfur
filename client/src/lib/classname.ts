export const cls = (...classNames: (string | boolean)[]) => classNames.filter(className => !!className).join(" ");
