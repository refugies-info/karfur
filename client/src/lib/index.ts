export const jsUcfirst = (string: string) =>
  string &&
  string.length > 1 &&
  string.charAt(0).toUpperCase() + string.slice(1, string.length);

export const jsLcfirst = (string: string) =>
  string &&
  string.length > 1 &&
  string.charAt(0).toLowerCase() + string.slice(1, string.length);

export const limitNbCaracters = (string: string, nbCaractersMax: number) =>
  string.substring(0, Math.min(string.length, nbCaractersMax)) +
  (string.length > nbCaractersMax ? "..." : "");

export const removeAccents = (str = "") => {
  var accents =
    "ÀÁÂÃÄÅàáâãäåÒÓÔÕÕÖØòóôõöøÈÉÊËèéêëðÇçÐÌÍÎÏìíîïÙÚÛÜùúûüÑñŠšŸÿýŽž";
  var accentsOut =
    "AAAAAAaaaaaaOOOOOOOooooooEEEEeeeeeCcDIIIIiiiiUUUUuuuuNnSsYyyZz";
  const chars = str.split("");
  let i, x;
  for (i = 0; i < chars.length; i++) {
    if ((x = accents.indexOf(chars[i])) !== -1) {
      chars[i] = accentsOut[x];
    }
  }
  return chars.join("");
};

// rules for password check :
// nbNumUpper = nbNumeric + nbUpper
// score 0 (cannot set password): length < 5
// score 0.5 : 7>length>=5 + nbNumUpper = 1 or 2 or0
// score 1.5 : 7>length >= 5 + nbNumUpper>=3 OR length>=7 + nbNumUpper<3
// score 2.5 : length >= 7 + nbNumUpper>=3
// score 3.5 : length >= 7 + nbNumUpper>=3 + at least 1 special caractere

// make sure to use same function in backend
export const computePasswordStrengthScore = (
  password: string
): { score: number } => {
  if (password.length < 5) {
    return { score: 0 };
  }

  const numerics = password.match(/\d+/g)?.join("");
  const nbNumeric = numerics ? numerics.length : 0;

  const upperCase = password.match(/[A-Z]/g)?.join("");
  const nbUpperCase = upperCase ? upperCase.length : 0;

  if (password.length < 7) {
    if (nbUpperCase + nbNumeric < 3) {
      return { score: 1 };
    }
    return { score: 2 };
  }

  if (nbNumeric + nbUpperCase < 3) {
    return { score: 2 };
  }
  const specialCar = password.match(/[!@#$&*]/g)?.join("");
  const nbSpecialCar = specialCar ? specialCar.length : 0;
  if (nbSpecialCar > 0) {
    return { score: 4 };
  }

  return { score: 3 };
};
