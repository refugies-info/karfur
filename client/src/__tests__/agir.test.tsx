import { wrapWithProvidersAndRender } from "../../jest/lib/wrapWithProvidersAndRender";
import agir from "../pages/agir";
import { initialMockStore } from "__fixtures__/reduxStore";
import { act, ReactTestRenderer } from "react-test-renderer";
import "jest-styled-components";

jest.mock("next/router", () => require("next-router-mock"));

describe("agir", () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  let component: ReactTestRenderer;

  it("renders traduire", () => {
    window.scrollTo = jest.fn();
    act(() => {
      component = wrapWithProvidersAndRender({
        Component: agir,
        reduxState: {
          ...initialMockStore,
        },
        compProps: {},
      });
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
});
