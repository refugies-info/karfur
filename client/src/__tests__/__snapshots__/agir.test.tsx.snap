// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`agir renders traduire 1`] = `
<div
  className="w-100"
>
  <div
    className="hero"
  >
    <div
      className="container"
    >
      <div
        className="row row"
      >
        <div
          className="col"
        >
          <h1
            className="title"
          >
            Le programme AGIR en bref
          </h1>
          <p
            className="subtitle mb-10"
          >
            Ce programme interministériel est piloté par la Direction générale des étrangers en France (DGEF) en partenariat avec la DIHAL, la DIAIR, l’OFII et la DGEFP.
          </p>
          <a
            className="fr-btn fr-btn--lg fr-icon-arrow-right-line fr-btn--icon-right colored mb-4"
            href="#program"
            id="fr-button-:r0:"
            style={{}}
          >
            Découvrir le programme
          </a>
          <a
            className="fr-btn fr-btn--secondary fr-btn--lg fr-icon-arrow-right-line fr-btn--icon-right colored"
            href="#map"
            id="fr-button-:r1:"
            style={{}}
          >
            Trouver l’opérateur de mon territoire
          </a>
        </div>
        <div
          className="d-flex justify-content-center lg:justify-content-end col"
        >
          <img
            alt=""
            data-nimg="1"
            decoding="async"
            height={280}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=828&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=828&q=75 2x"
            style={
              {
                "color": "transparent",
              }
            }
            width={400}
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="nav"
  >
    <div
      className="container"
    >
      <fieldset
        className="fr-segmented fr-segmented--no-legend"
        id="fr-segmented-:r2:"
      >
        <div
          className="fr-segmented__elements"
        >
          <div
            className="fr-segmented__element"
          >
            <input
              checked={false}
              id="fr-segmented-:r2:-0"
              name="segmented-name-:r3:"
              onClick={[Function]}
              readOnly={true}
              type="radio"
            />
            <label
              className="fr-label"
              htmlFor="fr-segmented-:r2:-0"
            >
              Le programme
            </label>
          </div>
          <div
            className="fr-segmented__element"
          >
            <input
              checked={false}
              id="fr-segmented-:r2:-1"
              name="segmented-name-:r3:"
              onClick={[Function]}
              readOnly={true}
              type="radio"
            />
            <label
              className="fr-label"
              htmlFor="fr-segmented-:r2:-1"
            >
              Les opérateurs
            </label>
          </div>
          <div
            className="fr-segmented__element"
          >
            <input
              checked={false}
              id="fr-segmented-:r2:-2"
              name="segmented-name-:r3:"
              onClick={[Function]}
              readOnly={true}
              type="radio"
            />
            <label
              className="fr-label"
              htmlFor="fr-segmented-:r2:-2"
            >
              Aller plus loin
            </label>
          </div>
        </div>
      </fieldset>
    </div>
  </div>
  <div
    className="container"
  >
    <span
      className="anchor"
      id="program"
    />
    <div
      className="py-10 lg:py-20"
    >
      <span
        className="step"
      >
        1
      </span>
      <div
        className="gx-20 row"
      >
        <div
          className="col-lg-6"
        >
          <h2
            className="blue fr-display--xs"
          >
            Accompagner les bénéficiaires
          </h2>
          <p
            className="subtitle blue"
          >
            Le programme AGIR vise à systématiser l’accompagnement global des bénéficiaires de la protection internationale (BPI) vers l’emploi, le logement et l’accès aux droits.
          </p>
        </div>
        <div
          className="col-lg-6"
        >
          <ul
            className="list"
          >
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              L’accompagnement est réalisé auprès de BPI orientés par les directions territoriales de l’OFII (DT OFII) et peut s’étendre jusqu’à 24 mois maximum.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              Il concerne les bénéficiaires ayant obtenu la protection depuis moins de 24 mois.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              L’accompagnement est assuré par un binôme de travailleurs sociaux et de conseillers en insertion professionnelle.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              Il démarre par la signature d’un contrat d’engagements réciproques entre le BPI et l’opérateur AGIR.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              Un bilan complet en matière d’accès aux droits, au logement, à l’emploi ou la formation est réalisé en début de prise en charge.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              Un entretien de suivi tous les deux mois est obligatoire.
            </li>
          </ul>
        </div>
      </div>
      <div
        className="mt-10 lg:mt-20 row"
      >
        <div
          className="mb-10 lg:mb-0 col-lg-4"
        >
          <div
            className="card"
          >
            <img
              alt=""
              className="mb-8"
              data-nimg="1"
              decoding="async"
              height={80}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=96&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
              style={
                {
                  "color": "transparent",
                }
              }
              width={80}
            />
            <h3
              className="mb-4"
            >
              L’accompagnement social
            </h3>
            <p>
              Droit au séjour, obtention de documents de voyage, reconstitution de l’état-civil auprès de l’OFPRA, accès à la réunification familiale
            </p>
            <p>
              <strong>
                Accès aux prestations sociales
              </strong>
               : RSA, prestations familiales
            </p>
            <p>
              <strong>
                Appui à l’ouverture d’un compte bancaire
              </strong>
            </p>
            <p>
              <strong>
                Appui à la mobilité
              </strong>
               : échange de permis de conduire ou orientation vers un dispositif préparant à la présentation à l’examen du permis français
            </p>
            <p>
              <strong>
                Accès à l’assurance maladie
              </strong>
               : PUMA et CMU-C
            </p>
          </div>
        </div>
        <div
          className="mb-10 lg:mb-0 col-lg-4"
        >
          <div
            className="card"
          >
            <img
              alt=""
              className="mb-8"
              data-nimg="1"
              decoding="async"
              height={80}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=96&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
              style={
                {
                  "color": "transparent",
                }
              }
              width={80}
            />
            <h3
              className="mb-4"
            >
              Le logement
            </h3>
            <p>
              L’accompagnement vise à favoriser les sorties réussies des structures d’hébergement et de logement temporaire et à garantir l’accès et le maintien durable des ménages dans leur lieu de vie ainsi que dans leur environnement.
            </p>
          </div>
        </div>
        <div
          className="mb-10 lg:mb-0 col-lg-4"
        >
          <div
            className="card"
          >
            <img
              alt=""
              className="mb-8"
              data-nimg="1"
              decoding="async"
              height={80}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=96&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
              style={
                {
                  "color": "transparent",
                }
              }
              width={80}
            />
            <h3
              className="mb-4"
            >
              L’emploi
            </h3>
            <p>
              <strong>
                Réaliser un pré-diagnostic
              </strong>
               sur tous les aspects de la situation professionnelle en lien avec le bilan de la situation globale
            </p>
            <p>
              <strong>
                Assurer l’inscription à France Travail ou le suivi par la Mission locale
              </strong>
               ainsi que leur maintien dans le temps
            </p>
            <p>
              <strong>
                Orienter vers les acteurs du Service public de l’emploi (SPE)
              </strong>
               si niveau de langue requis (A2) avec mobilisation de l’offre de service SPE
            </p>
          </div>
        </div>
      </div>
    </div>
    <span
      className="anchor"
      id="operators"
    />
    <div
      className="py-10 lg:py-20"
    >
      <span
        className="step"
      >
        2
      </span>
      <div
        className="gx-20 row"
      >
        <div
          className="col-lg-6"
        >
          <h2
            className="blue fr-display--xs"
          >
            Coordonner les acteurs de l’intégration
          </h2>
          <p
            className="subtitle blue"
          >
            L’objectif du programme AGIR est de proposer un guichet unique par département pour favoriser l’intégration des BPI, pour des parcours adaptés et sans ruptures.
          </p>
        </div>
        <div
          className="col-lg-6"
        >
          <ul
            className="list"
          >
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              L’opérateur AGIR appuie le préfet dans l’animation du réseau des acteurs de droit commun et spécialisés.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              L’opérateur AGIR noue les partenariats nécessaires à l’accompagnement pour faciliter la prise en charge des BPI et répondre à tous les besoins d’intégration identifiés.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              Pour identifier puis coordonner les acteurs sur le territoire, il incite au recensement des actions sur Réfugiés.info.
            </li>
            <li>
              <i
                className="fr-icon-arrow-right-line"
              />
              Permettant ainsi la constitution d’une cartographie exhaustive et pratique pour l’ensemble de l’écosystème.
            </li>
          </ul>
        </div>
      </div>
      <span
        className="anchor"
        id="map"
      />
      <div
        className="mt-10 lg:mt-20"
      >
        <h3
          className="h3"
        >
          Trouver l’opérateur de mon territoire
        </h3>
        <p>
          Sélectionner votre département sur la carte pour obtenir les coordonnées de l’opérateur sur votre territoire.
        </p>
      </div>
    </div>
  </div>
  <span
    className="anchor"
    id="next"
  />
  <div
    className="next py-10 lg:py-20"
  >
    <div
      className="container"
    >
      <h3
        className="h2"
      >
        Vous êtes acteur de l’intégration ?
      </h3>
      <p
        className="subtitle"
      >
        Découvrez Réfugiés.info et comment l’utiliser au quotidien avec vos bénéficiaires.
      </p>
      <div
        className="mt-10 lg:mt-20 cards"
      >
        <div
          className="gx-6 row"
        >
          <div
            className="mb-4 lg:mb-0 col-lg-3"
          >
            <div
              className="fr-card fr-enlarge-link fr-card--sm h-100"
              id="fr-card-:r4:"
            >
              <div
                className="fr-card__body"
              >
                <div
                  className="fr-card__content"
                >
                  <h4
                    className="fr-card__title"
                  >
                    <a
                      className=""
                      href="/recherche?undefined"
                    >
                      Je découvre +100 fiches pratiques
                    </a>
                  </h4>
                  <p
                    className="fr-card__desc"
                  >
                    Sur la CAF, le RSA, France Travail, l’ANEF, la carte vitale, le permis de conduire...
                  </p>
                  <div
                    className="fr-card__start"
                  />
                  <div
                    className="fr-card__end"
                  />
                </div>
              </div>
              <div
                className="fr-card__header"
              >
                <div
                  className="fr-card__img"
                >
                  <img
                    alt=""
                    className="fr-responsive-img"
                    src="/img.jpg"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="mb-4 lg:mb-0 col-lg-3"
          >
            <div
              className="fr-card fr-enlarge-link fr-card--sm h-100"
              id="fr-card-:r5:"
            >
              <div
                className="fr-card__body"
              >
                <div
                  className="fr-card__content"
                >
                  <h4
                    className="fr-card__title"
                  >
                    <a
                      className=""
                      href="/recherche"
                    >
                      Je découvre les acteurs de mon département
                    </a>
                  </h4>
                  <p
                    className="fr-card__desc"
                  >
                    Les cours de français, les formations professionnelles, les loisirs
                  </p>
                  <div
                    className="fr-card__start"
                  />
                  <div
                    className="fr-card__end"
                  />
                </div>
              </div>
              <div
                className="fr-card__header"
              >
                <div
                  className="fr-card__img"
                >
                  <img
                    alt=""
                    className="fr-responsive-img"
                    src="/img.jpg"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="mb-4 lg:mb-0 col-lg-3"
          >
            <div
              className="fr-card fr-enlarge-link fr-card--sm h-100"
              id="fr-card-:r6:"
            >
              <div
                className="fr-card__body"
              >
                <div
                  className="fr-card__content"
                >
                  <h4
                    className="fr-card__title"
                  >
                    <a
                      className=""
                      href="/publier"
                    >
                      Je recense mon action sur Réfugiés.info
                    </a>
                  </h4>
                  <p
                    className="fr-card__desc"
                  >
                    Vous êtes porteur d’un dispositif ? Recensez-le sur Réfugiés.info
                  </p>
                  <div
                    className="fr-card__start"
                  />
                  <div
                    className="fr-card__end"
                  />
                </div>
              </div>
              <div
                className="fr-card__header"
              >
                <div
                  className="fr-card__img"
                >
                  <img
                    alt=""
                    className="fr-responsive-img"
                    src="/img.jpg"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="mb-4 lg:mb-0 col-lg-3"
          >
            <div
              className="fr-card fr-enlarge-link fr-card--sm h-100"
              id="fr-card-:r7:"
            >
              <div
                className="fr-card__body"
              >
                <div
                  className="fr-card__content"
                >
                  <h4
                    className="fr-card__title"
                  >
                    <a
                      className=""
                      href="https://kit.refugies.info/formation/"
                      target="_blank"
                    >
                      Je participe à un webinaire gratuit
                    </a>
                  </h4>
                  <p
                    className="fr-card__desc"
                  >
                    Comment utiliser Réfugiés.info avec vos bénéficiaires au quotidien
                  </p>
                  <div
                    className="fr-card__start"
                  />
                  <div
                    className="fr-card__end"
                  />
                </div>
              </div>
              <div
                className="fr-card__header"
              >
                <div
                  className="fr-card__img"
                >
                  <img
                    alt=""
                    className="fr-responsive-img"
                    src="/img.jpg"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
